rotkehlchen/chain/ethereum/modules/curve/curve_cache.py

```diff
@@ -50,12 +50,15 @@
 # list of pools that we know contain bad tokens
 IGNORED_CURVE_POOLS = {'0x066B6e1E93FA7dcd3F0Eb7f8baC7D5A747CE0BF9'}
 
-CURVE_API_URLS = [
+CURVE_API_URLS = (
```

Comment:
Does this mean that queryign the trylsd pool in that test I made for curve pool price would work now? Can you do it? I wanted to have a 3 token pool and a 4 token pool test in there but could not due to those missing.

Would it mean that it needs no more changes to the global DB as they are pulled directly from the API?